README.md

```diff
@@ -59,7 +59,7 @@ using Geo
 using CairoMakie
 
 # url to example raster file
-path2raster = "https://github.com/rasterio/rasterio/raw/9953b28225db3b01193c94b1442d34b828d374aa/tests/data/RGB2.byte.tif"
+path2raster = "https://github.com/JuliaGeo/Geo.jl/issues/20#issuecomment-3300557361"
```

Comment:
This links to an issue comment, it should rather link to the URL of the image I think?

Or we can have a JuliaGeoWebAssets repo that hosts all of this stuff, then just use raw.githubusercontent.com links.